src/components/image-content-block/image-content-block.stories.mdx

```diff
@@ -0,0 +1,23 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+import { Image } from './image-content-block';
+import { Regular } from './image-content-block.stories';
+
+<Meta title="Components/Image" component={Image} />
+
+# Image
+
+The ContentBlock/Image Component can be used to display pictures with a description and an optional `attribution` tag in the bottom right corner.
+The image itself is passed as children. (e.g. as GatsbyImage Component, but always with `width: 100%;`for correct layouting).
```

Comment:
> but always with `width: 100%`

Is this something we expect or something we apply? It's not clear from the text.

Reply:
We expect it, I have changed it